docs-library/10x-FY18-Project-Results.md

```diff
@@ -0,0 +1,41 @@
+# FY18 10x Project Results
+
+## Phase One. **Investigation**
+
+**Government-wide User Testing Program** — Continued to next phase. [Final report here](https://docs.google.com/document/d/1HxCFhc27NguZwervykWLG8NMLAryWoPFQ1BiPKe-yAI/edit#heading=h.r9d3o4h63t06). Decision was made to continue this project at the $175k Discovery phase. It is clear that federal agencies face a host of barriers to conducting user research and testing. A government-wide resource helping unblock the most common barriers (such as PRA interpretation, user recruitment, a need for better design and research tools and resources, and an overall understanding of the value of research) would enable federal agencies to more effectively conduct user research and make smarter decisions when building or buying products. 
+
+**Qualitative Data Management** — Continued to next phase. [Final report here](https://docs.google.com/document/d/1InUpl7v3wa0v05JYCB8-atoDene9-Gzbz-ELY7OPVKY/edit#heading=h.7n91pqag6jmk). Decision was made to continue this project at the $175k Discovery phase. There’s much work to be done in this space, and the value is likely to offer significant gain in both how we communicate with citizens searching for information on government services, as well as improving the rate at which we incorporate valuable public feedback into our services and products. 
+
+**Human-Centered Design of Shared Services** — Continued to next phase. [Final report here](https://docs.google.com/document/d/14CTh8Jxrhqo6LnvxihxzCUTQ5r02pSnTrMCFCFgoI38/edit#heading=h.lcvy8u5agord). Decision was made to continue this project at the $175k Discovery phase. Transitioning to using shared services is usually challenging for agencies and we see substantial opportunity for TTS to help make shared services across the government best serve the agencies’ missions.
+
+**Open Source Translation Services** — Continued to next phase. [Final report here](https://docs.google.com/document/d/1TkOpvhNiMdkT73dv5-oefKPSSuomfGUY3DIeq9hx5vA/edit) and presentation [here](https://docs.google.com/presentation/d/1F-2lkGwLTJqIxkUvugBhqofBKbRtcArdX6i8mm0n0uY/edit#slide=id.p). Decision was made to continue this project at the $175k Discovery phase. There is an opportunity to help consolidate and automate existing translation workflows using shared translations and terminology across the federal government.
```

Comment:
Recommend making the link to the presentation include the word presentation. Users of assistive technology will often have their screen readers read all the links off the page and you don't want them to just here a bunch of "here, link; here, link" all the way down. So maybe more like 

```[Final report](https://docs.google.com/document/d/1TkOpvhNiMdkT73dv5-oefKPSSuomfGUY3DIeq9hx5vA/edit) and [presentation](https://docs.google.com/presentation/d/1F-2lkGwLTJqIxkUvugBhqofBKbRtcArdX6i8mm0n0uY/edit#slide=id.p)```

Reply:
same for other instances on this doc.